test/unit/org/apache/cassandra/audit/AuditLoggerTest.java

```diff
@@ -59,17 +59,17 @@
 public class AuditLoggerTest extends CQLTester
 {
     @BeforeClass
-    public static void setUp()
+    public static void setUp() throws IOException
     {
-        AuditLogOptions options = new AuditLogOptions();
+        AuditLogOptions options = getBaseAuditLogOptions();
         options.enabled = true;
         options.logger = new ParameterizedClass("InMemoryAuditLogger", null);
         DatabaseDescriptor.setAuditLoggingOptions(options);
         requireNetwork();
     }
 
     @Before
-    public void beforeTestMethod()
+    public void beforeTestMethod() throws IOException
     {
         AuditLogOptions options = new AuditLogOptions();
```

Comment:
Should this and the other instance like it also use getBadAuditLogOptions for consistency?

Reply:
I actually started that refactor at first and then stopped because I wasn't sure whether consistency trumped minimizing changes. Happy to do that work if we all agree it's the right thing to do

Reply:
I definitely think both consistency and having one way to do it are the way to go here.

Reply:
+1

Reply:
PR is updated with the change for consistency